Resources/Prototypes/Entities/Structures/Wallmounts/door_buttons.yml

```diff
@@ -0,0 +1,60 @@
+- type: entity
+  id: NetworkedButtonBase
+  abstract: true
+  placement:
+    mode: SnapgridCenter
+    snap:
+      - Wallmount
+  components:
+    - type: WallMount
+      arc: 360
+    - type: Clickable
+    - type: InteractionOutline
+    - type: Physics
+    - type: Sprite
+      sprite: Structures/Wallmounts/switch.rsi
+      state: dead
+    - type: UseDelay
+      delay: 0.5 # prevent light-toggling auto-clickers.
+    - type: ApcPowerReceiver
+    - type: ExtensionCableReceiver
+    - type: DeviceList
+    - type: DeviceNetworkRequiresPower
+    - type: AccessReader
+      access: [ [ "Engineering" ] ]
```

Comment:
this is primarily here because otherwise, DeviceList wouldn't work - but yeah, these buttons should either have the ability to have overridable access tags that are sent with the packet/preconfigured ones that a mapper can just drop down